src/lib/components/workflow/advanced-filter/index.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import StatusFilter from './status-filter.svelte';
+  import { searchAttributes } from '$lib/stores/search-attributes';
+  import { noop } from 'svelte/internal';
+  import IntFilter from './int-filter.svelte';
+  import KeywordFilter from './keyword-filter.svelte';
+  import DatetimeFilter from './datetime-filter.svelte';
+  import CustomButton from '$lib/holocene/custom-button.svelte';
+  import BooleanFilter from './boolean-filter.svelte';
+
+  export let attribute: string | number;
+  export let value: string = '';
+  export let conditional: string = '';
+
+  export let removeFilter: () => void = () => noop;
```

Comment:
Shouldn't this be `export let removeFilter: () => void = noop;`?

Reply:
yup